src/lens/Quoter.sol

```diff
@@ -1,9 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 pragma solidity ^0.8.20;
 
-import {Hooks} from "@uniswap/v4-core/src/libraries/Hooks.sol";
 import {TickMath} from "@uniswap/v4-core/src/libraries/TickMath.sol";
-import {IHooks} from "@uniswap/v4-core/src/interfaces/IHooks.sol";
 import {IUnlockCallback} from "@uniswap/v4-core/src/interfaces/callback/IUnlockCallback.sol";
```

Comment:
and this one

Reply:
and CurrencyLibrary here: https://github.com/Uniswap/v4-periphery/blob/3e343f22c23dac08cedce57b92aff13d58d4a08d/src/V4Router.sol#L7